practical/groovy/background_subtract_batch.groovy

```diff
@@ -0,0 +1,194 @@
+/*
+ * -----------------------------------------------------------------------------
+ *  Copyright (C) 2018 University of Dundee. All rights reserved.
+ *
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * ------------------------------------------------------------------------------
+ */
+
+/*
+ * This Groovy script uses ImageJ to Subtract Background.
+ * Use this script in the Scripting Dialog of Fiji (File > New > Script).
+ * Select Groovy as language in the Scripting Dialog.
+ * Error handling is omitted to ease the reading of the script but this
+ * should be added if used in production to make sure the services are closed
+ * Information can be found at
+ * https://docs.openmicroscopy.org/latest/omero5/developers/Java.html
+ */
+
+import java.io.File
+import java.util.ArrayList
+import java.lang.reflect.Array
+
+// OMERO Dependencies
+import omero.gateway.Gateway
+import omero.gateway.LoginCredentials
+import omero.gateway.SecurityContext
+import omero.gateway.facility.BrowseFacility
+import omero.gateway.facility.DataManagerFacility
+import omero.gateway.model.DatasetData
+import omero.log.SimpleLogger
+
+import ome.formats.importer.ImportConfig
+import ome.formats.importer.OMEROWrapper
+import ome.formats.importer.ImportLibrary
+import ome.formats.importer.ImportCandidates
+import ome.formats.importer.cli.ErrorHandler
+import ome.formats.importer.cli.LoggingImportMonitor
+
+import loci.formats.in.DefaultMetadataOptions
+import loci.formats.in.MetadataLevel
+
+import ij.IJ
+
+// Setup
+// =====
+
+// OMERO Server details
+HOST = "outreach.openmicroscopy.org"
+PORT = 4064
+group_id = "-1"
+// parameters to edit
+dataset_id = "1001"
```

Comment:
Here, the ``dataset_id`` is in quotation marks, unlike in the previous two scripts